### PR TITLE
Add missing nbs fields to enable use of StreamReactor in the NUC side

### PR DESCRIPTION
**Pull request by JohanneMontano**

### CODE REVIEW

**Review by Claegtun — CHANGES_REQUESTED** [review]
Overall, this needs to be tested on the robot first, ideally with KeyboardWalk, etc. such that we know that no functionality has accidently been lost and that no new bugs spring up.

Therefore, this review may only be ephemeral since more changes may be made henceforth.

Lastly, this should be merged at the same time as the corresponding HardwareIO branch.

**Review by ysims — APPROVED** [review]

File: NUSense/Core/Src/usb/PacketHandler.hpp [hunk 1]
```diff
@@ -33,21 +33,23 @@ namespace usb {
                     && (rx_buffer.data[(rx_buffer.front + 1) % RX_BUF_SIZE] == (char) 0x98)
                     && (rx_buffer.data[(rx_buffer.front + 2) % RX_BUF_SIZE] == (char) 0xA2)) {
 
-                    pb_length = static_cast<uint16_t>(rx_buffer.data[(rx_buffer.front + 3) % RX_BUF_SIZE] << 8)
-                                | static_cast<uint16_t>(rx_buffer.data[(rx_buffer.front + 4) % RX_BUF_SIZE]);
+                    pb_length = static_cast<uint32_t>(rx_buffer.data[(rx_buffer.front + 3) % RX_BUF_SIZE] << 0)
```
[the remark this answers is not in the document]
Comment (Claegtun) [reply NUSense/Core/Src/usb/PacketHandler.hpp:36]: Actually, never mind; it is fine as it is.

File: NUSense/Core/Src/test_hw.hpp [hunk 2]
```diff
@@ -131,5 +131,4 @@ namespace test_hw {
 #endif
 
 }  // namespace test_hw
```
Comment (Claegtun) [inline NUSense/Core/Src/test_hw.hpp:133]:
```suggestion
}  // namespace test_hw

```
Just quickly change this for me.

**Review by Claegtun — CHANGES_REQUESTED** [review]

**Review by Claegtun — APPROVED** [review]
Seems good.

**Comment by Claegtun** [comment]
However, we may want to wait until the other PR is ready to merge as well.

**Comment by ysims** [comment]
This may as well be merged, none of the NUSense stuff on here works with main on NUbots/NUbots and the branch on that side is stream reactor now anyway.

**Pull request merged**